File: Java/Dado.java
import java.util.Scanner;

public class Dado {
    private int valorDado;

    public int getValorDado() {
        return valorDado;
    }

    public void setValorDado(int valorDado) {
        this.valorDado = valorDado;
    }

    public int generarNumero() {
        valorDado = (int) (Math.random() * 6) + 1;
        return valorDado;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        Dado dado1 = new Dado();
        System.out.println("Ingrese la cantidad de tiros");
        int cantidadTiros = scanner.nextInt();
        for (int i = 1; i <= cantidadTiros; i++) {
            System.out.println("El tiro " + i + " sale: " + dado1.generarNumero());
        }
    }
}

File: Python/Dado.py
from random import randint


class Dado():
    __valorDado = None

    def getValorDado(self):
        return self.__valorDado

    def setValorDado(self, valorDado):
        self.__valorDado = valorDado

    def generarNumero(self):
        self.__valorDado = randint(1, 6)
        return self.__valorDado


def main():
    miDado = Dado()
    cantidadTiros = int(input("Ingrese la cantidad de tiros: "))
    for i in range(cantidadTiros):
        print("El tiro ", i+1, " sale; ", miDado.generarNumero())


if __name__ == '__main__':
    main()
